core/java/test/org/openda/exchange/timeseries/TimeSeriesTest.java

```diff
@@ -490,30 +487,30 @@ public static void testTimeSeriesSelection_2() {
       assertEquals("values2", "[0.1, 0.2, 0.2, 0.3, 999.0]", Arrays.toString(values2));
    }
 
-   public static void testTimeSeriesSelection_3() {
-      double times[] = { 0.0, 1.0, 2.0, 3.0, 4.0 };
-      double values[] = { 0.0, 0.1, 0.2, 0.3, 0.4 };
+   public void testTimeSeriesSelection_3() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The JUnit 3 test method name 'testTimeSeriesSelection_3' doesn't match 'test[A-Z0-9][a-zA-Z0-9]*'](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=5102854)